packages/hackathon/src/scenes/ProjectsScene/ProjectsScene.tsx

```diff
@@ -63,6 +67,23 @@ export const ProjectsScene: FC<ProjectSceneProps> = () => {
     if (hackathon) dispatch(lockProjects(false, hackathon._id))
   }
 
+  useEffect(() => {
+    if (hackathon && hacker) {
+      const started = hackathon.judging_starts?.getTime() < new Date().getTime()
+      setJudgingStarts(
+        `Judging start${started ? 'ed' : 's'} ${zonedLocaleDate(
```

Comment:
`started ? 'ed' : 's'` is dead code given conditional rendering at line 97. 
With that said, I say we remove the conditional rendering at line 97 and always show when project judging will begin.

Also need to add `at` so it reads `Judging start(s)(ed) at date).

Reply:
I am undecided where judging start time should show in the final implementation, so I'm only showing it for now when it has an impact on the project board. If I added "at" to be grammatically correct, I'd also need to add "on".

So I'm going with a colon instead.